### PR TITLE
use paragraphs for top level items, not lists JLBP-4

### PR DESCRIPTION
**Pull request by elharo**
@garrettjonesgoogle

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
I think removing the bullets on this one seems fine, but it could do with combining paragraphs more.

File: docs/JLBP-4.md [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
 library's current major version is permanently locked to the version
-you initially exposed, and you won't be able to upgrade without making a
+you initially exposed, and you cannot upgrade without making a
 breaking change to your users.
 
-- If you only use the unstable feature in your implementation, then each minor
+If you only use the unstable feature in your implementation, then each minor
 or patch version of your library requires a very specific version of
-your dependency, and it is unsafe for your users to upgrade your
+your dependency; and it is unsafe for your users to upgrade your
```
[the remark this answers is not in the document]
Comment (elharo) [reply docs/JLBP-4.md]: done

File: docs/JLBP-4.md [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
-  avoid doing so.
-
-  - Depending on unstable features between submodules of a single library is
-    acceptable, provided that users can easily force their build system to use
-    compatible versions of the submodules. Some strategies for library owners
-    include:
-    
-    - Using the same version number for all submodules in a library
-    - Providing a [BOM](http://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies)
+Do not depend on libraries that are pre-1.0 or on unstable features in
```
Comment (garrettjonesgoogle) [inline docs/JLBP-4.md]: I would combine this paragraph with the next two (the ones starting with "Unstable")

File: docs/JLBP-4.md [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
+Unstable libraries allow breaking changes to their
+public APIs within the same major version. For libraries following semantic
+versioning, this means libraries with a 0.x.y version. See [JLBP-3](JLBP-3.md)
+for more details on semantic versioning.
+
+Unstable features within an otherwise stable library
+are marked with an annotation such as `@Beta`. See
+[JLBP-3](JLBP-3.md) for more details on annotating unstable features.
+
+If your library depends on an unstable library or feature which
```
Comment (garrettjonesgoogle) [inline docs/JLBP-4.md]: I would combine this paragraph with the next two (the ones starting with "If")

File: docs/JLBP-4.md (hunk 2 above)
Comment (elharo) [reply docs/JLBP-4.md]: done

File: docs/JLBP-4.md (hunk 3 above)
Comment (elharo) [reply docs/JLBP-4.md]: done

**Comment by garrettjonesgoogle** [comment]
I'm not seeing any new commits since my last comments

**Pull request merged**